report_viewer/sampleData.json

```
@@ -0,0 +1,10921 @@
+{
+  "resourceType": "Bundle",
+  "id": "bc3f7243-8a9c-4919-b5c9-47a71529df26",
+  "meta": {
+    "lastUpdated": "2022-03-07T21:52:02.071+00:00"
+  },
+  "type": "searchset",
+  "total": 1,
+  "link": [ {
+    "relation": "self",
+    "url": "https://fhir.localtest.me/fhir/Bundle?_pretty=true"
+  } ],
+  "entry": [
+    {
```

I _think_ this curly brace is out of alignment with the one that starts the next Bundle
```suggestion
  {
```